flutter_local_notifications/lib/src/platform_specifics/android/active_notification.dart

```diff
@@ -0,0 +1,20 @@
+class ActiveNotification {
+  const ActiveNotification(
+    this.id,
+    this.channelId,
+    this.title,
+    this.body,
+  );
+
+  /// The notification's id.
+  final int id;
+
+  /// The notification channel's id - Returned only on Android 8.0+
```

Comment:
nit: I've been putting things similar _Returned only on Android 8.0+_ in a separate paragraph as part of following these guidelines https://dart.dev/guides/language/effective-dart/documentation#do-start-doc-comments-with-a-single-sentence-summary

Reply:
ok sure